client/components/Sidebar.js

```diff
@@ -58,14 +58,14 @@ const GenericItem = ({ href, active, children, ...props }) => <Box
 	</Box>
 </Box>;
 
-const NavigationItem = ({ permissionGranted, pathGroup, pathSection, icon, label, currentPath }) => {
+const NavigationItem = ({ permissionGranted, pathGroup, pathSection, icon, label, currentPath, tag }) => {
 	const params = useMemo(() => ({ group: pathGroup }), [pathGroup]);
 	const path = useRoutePath(pathSection, params);
 	const isActive = path === currentPath || false;
 	if (permissionGranted && !permissionGranted()) { return null; }
 	return <Sidebar.GenericItem active={isActive} href={path} key={path}>
 		{icon && <Icon name={icon} size='x20' mi='x4'/>}
-		<Box withTruncatedText fontScale='p1' mi='x4' color='info'>{label}</Box>
+		<Box withTruncatedText fontScale='p1' mi='x4' color='info'>{label} {tag && <Tag style={{ display: 'inline', backgroundColor: '#000', color: '#FFF', marginLeft: 4 }}>{tag}</Tag>}</Box>
```

Comment:
Should not this tag style be a specific class?

Reply:
Tag component needs to be changed on fuselage because its not accepting className.